src/locales/index.js

```diff
@@ -61,7 +47,4 @@ i18n.addResources('vi', namespace, viTranslations);
 i18n.addResources('zh', namespace, zhTranslations);
 i18n.addResources('zh_CN', namespace, zh_CNTranslations);
 
-i18n.setDefaultNamespace(namespace);
-i18n.changeLanguage('en');
-
 export default i18n;
```

Comment:
Usually we add `./src/locales` to `.gitignore` and don't commit this file to version control, is there a reason it is included here?

Reply:
At one point we ran into some problems when we didn't check it in (can't remember the exact details). It seems to work fine now though, so I've removed it from git.